Editor/Scripts/Settings/RosalinaSettings.cs

```diff
@@ -11,9 +11,14 @@
 [FilePath("ProjectSettings/RosalinaSettings.asset", FilePathAttribute.Location.ProjectFolder)]
 public class RosalinaSettings : ScriptableSingleton<RosalinaSettings>
 {
+    private const string DEFAULT_GENERATED_FOLDER = "Assets/Rosalina/Autogenerated";
+
     [SerializeField]
     private bool _isEnabled;
 
+    public string Namespace;
+    public string GenerateFolder = DEFAULT_GENERATED_FOLDER;
```

Comment:
Probably should be something like `GeneratedScriptsFolder` to make it more clear